example/middleware/hapi.js

```diff
@@ -3,40 +3,43 @@ const { graphqlHapi } = require('graphql-server-hapi');
 const { hapi: voyagerMiddleware } = require('graphql-voyager/middleware');
 const schema = require('../schema');
 
-const server = new Hapi.Server();
-
-server.connection({
+const server = new Hapi.Server({
   port: 3001
 });
 
-server.register([
-  {
-    register: graphqlHapi,
-    options: {
-      path: '/graphql',
-      graphqlOptions: {
-        schema,
-      },
-      route: {
-        cors: true
+const init = async () => {
```

Comment:
Node.js 6.x doesn't support async/await and I worried that generated js files contain them.
Can you please check that TS transpiles async/await?

Reply:
This example is based off of https://hapijs.com/tutorials

Reply:
And as far as I can tell this is not actually TS at all in the examples.  Just JS

Reply:
Looks like hapi supports only node v8+
https://github.com/hapijs/hapi/blob/master/package.json#L18
So not an issue.

Reply:
Correct